conf/machine/imx8mnevk.conf

```diff
@@ -0,0 +1,50 @@
+#@TYPE: Machine
+#@NAME: NXP i.MX8M Nano DDR4 Evaluation Kit
+#@SOC: i.MX8MM
+#@DESCRIPTION: Machine configuration for NXP i.MX8M Nano DDR4 EVK
+#@MAINTAINER:
+
+MACHINEOVERRIDES =. "mx8:mx8m:mx8mn:"
+
+require conf/machine/include/imx-base.inc
+require conf/machine/include/tune-cortexa53.inc
+
+MACHINE_FEATURES += " wifi bluetooth optee qca9377"
```

Comment:
Loose the space and add _pci_ if you want _pciutils_.

Reply:
> 
> Loose the space and add _pci_ if you want _pciutils_.

AFAIK _pci_ is not a relevant feature here, since i.MX8M Nano lacks the IP.

Reply:
The empty space is not need as `+=` adds it automatically.